Modules/Managers/PathManager.m

```diff
@@ -218,20 +195,48 @@ function menu_open_CB(obj,hObj,~)
         end
         function view_path(obj,~,~)
             obj.assert(~isempty(obj.paths),'No paths!')
-            f = figure('name',mfilename,'IntegerHandle','off','menu','none',...
-                'toolbar','none','visible','off','units','characters');
+            f = figure('name',[mfilename '(right click for more options)'],...
+                'IntegerHandle','off','menu','none','toolbar','none','units','characters');
             left = uicontrol(f,'style','listbox','units','normalized','string',{obj.paths.name},...
                 'position',[0 0 0.5 1],'callback',@obj.update_view,'value',1);
             right = uicontrol(f,'style','listbox','units','normalized','position',[0.51 0 0.49 1]);
+            c = uicontextmenu(f);
+            uimenu(c,'Label','Edit path','callback',{@obj.edit_path_view_path_CB,left});
+            uimenu(c,'Label','Delete path','callback',{@obj.delete_path_view_path_CB,left});
+            left.UIContextMenu = c;
+            right.UIContextMenu = c;
             left.UserData.right = right;
             obj.update_view(left);
-            f.Visible = 'on';
         end
         function update_view(obj,hObj,~)
+            % hObj -> left panel
             val = hObj.Value;
-            hObj.UserData.right.String = obj.get_string_instructions(obj.paths(val).instructions);
+            if obj.paths(val).alias
+                hObj.UserData.right.String = {'alias',sprintf('---> %s',obj.paths(val).instructions)};
+            else
+                hObj.UserData.right.String = strsplit(obj.paths(val).instructions,newline);
```

Comment:
If you aren't sure, this splits lines into a cell array, which is what String is expecting.